lib/diary.js

```diff
@@ -13,11 +25,35 @@ $(document).ready(function(){
       })
       let caloriesConsumed = 0
       totalCalories(caloriesConsumed)
+      hideDeleteButtons()
+      addCheckboxes()
+      addToMealTable()
     })
     .catch(function(error){
       console.error(error)
     })
-})
+}
+
+function collectTableSelection(mealId){
+  $.each($("#foodTable :checkbox:checked"), function(){
+    $.post("http://localhost:3000/api/v1/meals/" + mealId + "/foods/" + ($(this).closest("tr").find("td:eq(3)").children().attr("id")), function(data){
+    })
+    var result = $(this).closest("td").siblings().text()
+  })
+}
+
+function removeCheckedBoxes(){
+  $("#foodTable input:checkbox").prop("checked", false)
+}
+
+function addCheckboxes(){
```

Comment:
I'm glad this worked. I think it was how we were importing jQuery that was preventing us from acting on the table elements. Thanks for figuring this out!